app/controllers/provider_interface/application_choices_controller.rb

```diff
@@ -10,12 +10,22 @@ def index
     end
 
     def show
-      application_choice = GetApplicationChoicesForProvider.call(provider: current_user.provider)
-        .find(params[:application_choice_id])
-
       @application_choice = ApplicationChoicePresenter.new(application_choice)
     end
 
+    def respond
```

Comment:
As discussed offline, perhaps move into a `DecisionsController` to shadow the API controller structure.